src/createServer.js

```diff
@@ -1,3 +1,36 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { handleErrors } = require('./handleErrors');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const originalText = req.url.split('?').slice(0, 1).join('').slice(1);
```

Comment:
Use destructuring assignment to get the originalText and queryString from req.url.